internal/topo/node/sink_node.go

```diff
@@ -157,11 +157,12 @@ func (m *SinkNode) Open(ctx api.StreamContext, result chan<- error) {
 							for {
 								select {
 								case data := <-m.input:
-									if temp, processed := m.preprocess(data); processed {
+									var temp interface{}
```

Comment:
Actually, I prefer the previous style lol. What's the advantage of this link?

Reply:
`superfluous-else` removes the redundant `else` in the code.

What about revise code in this way:

```golang
if temp, processed := m.preprocess(data); processed {
    data = temp
} else {
    break
}
```

Reply:
Looks better.